Source/AwsCommonRuntimeKit/auth/signing/SigningConfig.swift

```diff
@@ -32,11 +36,15 @@ public struct SigningConfig {
                 shouldSignHeader: ShouldSignHeader? = nil,
                 signatureType: SignatureType = .requestHeaders,
                 signingAlgorithm: SigningAlgorithmType = .signingV4,
-                configType: SigningConfigType = .aws) {
+                configType: SigningConfigType = .aws,
+                allocator: Allocator = defaultAllocator) {
+
         self.credentials = credentials
         self.credentialsProvider = credentialsProvider
         self.expiration = expiration
         self.date = date
+        self._service = AWSStringByteCursor(service, allocator: allocator)
+        self._region = AWSStringByteCursor(region, allocator: allocator)
```

Comment:
This will be fixed by Fix CStruct PR. Will most probably nuke the AWSStringByteCursor class.